src/cluster.h

```diff
@@ -214,6 +214,12 @@ typedef struct clusterState {
     long long stats_pfail_nodes;    /* Number of nodes in PFAIL status,
                                        excluding nodes without address. */
     unsigned long long stat_cluster_links_buffer_limit_exceeded;  /* Total number of cluster links freed due to exceeding buffer limit */
+
+    /* Bit map for slots that are no longer claimed by the owner in cluster PING messages.
+     * During slot migration, the owner will stop claiming the slot after the ownership transfer.
+     * Set the bit corresponding to the slot when a node stops claiming the slot. This prevents
+     * spreading incorrect information (that source still owns the slot) using UPDATE messages. */
+    unsigned char owner_not_owning_slot[CLUSTER_SLOTS/8];
```

Comment:
```suggestion
    unsigned char owner_not_claiming_slot[CLUSTER_SLOTS/8];
```
Is probably a bit more accurate.

Reply:
Considering this adds 2KiB per node, would it be possible to allocate the tombstone bitmap only when nonempty and clear the tombstones after some time so the bitmap can be freed again?

Tombstones are set only from direct messages and not via gossip, right? Would it be safe to clear a tombstone when another node claims a slot directly, i.e. not via gossip?

Reply:
This is not per node, there is a single bitmap for the entire cluster.

Reply:
Ah, this is in clusterState. I missed that.

Reply:
Renamed the bitmap to `owner_not_claiming_slot`. 

Will leave the 2KiB statically allocated in clusterState.